src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/LoanAvailability.js

```diff
@@ -33,6 +33,7 @@ export class LoanAvailability extends Component {
             <b>
               {DateTime.fromISO(circulation.next_available_date).toLocaleString(
                 {
+                  locale: 'en-GB',
```

Comment:
👍 
any other place where this should be applied? Shall we put this somewhere reusable and change all occurrences? (maybe another PR)

Reply:
Added to CERNDocumentServer/cds-ils#17.